### ISSUE TITLE
Not working in Chrome

### ISSUE THREAD

**Issue by EmilMoe** [tmmdata/chartist-plugin-tooltip#146]
Hi..

I tried your demo at http://gionkunz.github.io/chartist-js/plugins.html but the tooltip is not showing in Chrome. Works in Firefox though.

**Comment by stklcode** [comment]
Just saw this... Works fine for me in Chrome 64 and 65 on Linux Desktop, so you probably should provide more details on your environment.

**Comment by EmilMoe** [comment]
Here's an output from https://www.whatsmybrowser.org/

~~~~
Operating system Windows NT 10.0
Javascript enabled Yes
Cookies enabled Yes
Flash version Not installed
Java version Not installed
Websockets supported Yes
Browser size 2543 x 1344
Screen size 2560 x 1440 (Retina)
Color depth 24 bit
Your full user agent string is:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.162 Safari/537.36
~~~~

**Comment by SimoneRiboldi** [comment]
I have the same issue:
Operating system Windows NT 10.0
Javascript enabled Yes
Cookies enabled Yes
Flash version 29.0.0
Java version Not installed
Websockets supported Yes
Browser size 1519 x 759
Screen size 1536 x 864 (Retina)
Color depth 24 bit

**Comment by duchuy** [comment]
i changed 
    <script type="text/javascript" src="chartist.min.js"></script>
to 
    <script type="text/javascript" src="chartist.js"></script>

and it worked. It seems like the minified version of chartist.js has some problem

https://github.com/tmmdata/chartist-plugin-tooltip/issues/127

**Comment by EmilMoe** [comment]
The demo works in Chrome on MacOS High Sierra

Version 65.0.3325.181 (Official Build) (64-bit)

**Comment by zeknoss** [comment]
I can confirm that plugin works only with non-minified version of Chartist.

**Comment by hanishassim** [comment]
Agreed, the minified version somewhat breaks the tooltip plugin, I've had this issue, and resolved it using non-minified version of Chartist.

Tested with Chrome (67.0.3396.99) on Windows 10
